contracts/test/mocks/apps/disputable/DisputableAppMock.sol

```diff
@@ -16,7 +16,7 @@ contract DisputableAppMock is DisputableAragonApp {
         initialized();
```

Comment:
I think we're missing one instance in the [`AgreementMock`](https://github.com/aragon/aragonOS/blob/next/contracts/test/mocks/apps/disputable/AgreementMock.sol#L5).

Reply:
Yes, if you can also clean up a few compiler warnings about this one it would be really helpful!